zerver/views/messages.py

```diff
@@ -910,9 +910,12 @@ def update_message_backend(request, user_profile,
         if content == "":
             raise JsonableError(_("Content can't be empty"))
         content = truncate_body(content)
-
+        ums = UserMessage.objects.filter(message=message.id)
```

Comment:
Yeah; I guess it's less important to me which choice we make here, but more that we highlight that there _is_ a choice here in the comment.  I don't think we have a current application where the decisions is important, but I can see that changing down the line.